source/locale/en/symbols.dic

```diff
@@ -167,6 +167,7 @@ _	line	most
 ®	registered	some
 ™	Trademark	some
 ©	Copyright	some
+℠ Servicemark some

```

Comment:
This is not going to work, even though automatic checks are not failing on this PR for whatever reason. The fields should be separated with tabs not spaces.

Reply:
interestingly I removed the space and inserted a tab and it tracked no change...Lemme update!

Reply:
Ok, I've added a tab where the space used to be! Thanks again!

Reply:
There are still spaces there not tabs.
```suggestion
℠	Servicemark	some

```